app/models/measure_collection.rb

```diff
@@ -10,6 +10,10 @@ def initialize(measures)
     @measures = measures.clone
   end
 
+  def exclude_supplementary_unit
+    self.class.new(measures.reject { |m| m.measure_type.description == 'Supplementary unit' })
```

Comment:
We 100% shouldn't use a changing description for this

Reply:
Yeh, I was not sure about it, but I could not find any info in the table `measure_types` and `measures` where the relevant info is stored (and not the description).